### ISSUE TITLE
Se debe definir una función para consultar el estado de un led

### ISSUE THREAD

**Issue by evolentini** [Nicomedes77/TDD_TP3#2]
Estas dos pruebas deberían utilizar una función que permita saber si un led esta encendido o apagado

https://github.com/Nicomedes77/TDD_TP3/blob/3bfad782d7a4582554bd9a379f015cb9259f1890/test/test_leds.c#L51-L69

**Comment by Nicomedes77** [comment]
Corregido en vs_B

**Comment by evolentini** [comment]
Para probar correctamente el funcionamiento de la función `ledIsOn()` se requieren dos pruebas: una con un led encendido y otra con un led apagado. Los mismo aplica a la función `ledIsOff()`, así que o se elimina una función o se aumentan dos pruebas (o se llaman a ambas funciones en cada prueba).

**Comment by Nicomedes77** [comment]
Ok Esteban. Para saber si entendí bien: en cada función debería "prender y apagar" el led para evaluar la función? por ejemplo, para la función de verificar encendido de un led deberia: apagarlo primero, encenderlo y luego verificarlo? estoy en lo correcto? lo mismo para la de verificar apagado.

Yo tuve en cuenta que la funcion de inicialización ya apaga todos los leds. Debería volver a apagarlos?

**Comment by evolentini** [comment]
La idea es que cada función debería devolver una vez `true` y una vez `false`, de lo contrario lo que dice la metodología es que lo podrías resolver devolviendo siempre un solo valor sin usar los parámetros para nada. Como ya sabes que los leds arrancan apagados es totalmente valido llamar a la función de consulta inmediatamente sin hacer nada antes. ¿Queda mas claro?

**Comment by Nicomedes77** [comment]
Ok. Ahora creo que entendí: evaluo la misma funcion `verifyOn()` en ambos test pero verificando que me devuelva un `true` en un caso y un `false` en el otro. esta bien?

**Comment by evolentini** [comment]
Correcto, esa una opción. La otra escribir cuatro pruebas. Pero en cualquier caso las dos funciones deben revolver los dos resultados posibles.
